recipe/meta.yaml

```diff
@@ -13,33 +13,32 @@ source:
 build:
   number: {{ build_number }}
 
-# A number of the requirements below will have two instances: one will include a specific version
-# pinning, while the other one will not. When building a version of ALE for use specifically within
-# ISIS, use the instances with the version pinned. For all other builds, use the unpinned instances.
-# (Confirm that every requirement has one and only one instance uncommented.)
-#
-# ISIS requires pinned versions for these dependencies to avoid conflicts with other dependencies.
-# Make sure to always confirm that version numbers in this file and version numbers in the
-# meta.yaml file within the ISIS3 repo match for each dependency before building this library
-# for use within ISIS, and always upload this special ISIS build with the "isis" tag.
 requirements:
   build:
-  - cmake>=3.10
-  - eigen
+    - cmake>=3.10
+    - eigen
+  host:
+    - pip
+    - python
+    - nlohmann_json
   run:
-  - eigen
-  - networkx
-  - numpy
-  - openblas
```

Comment:
Why is `openblas` still in the `environment.yml` but eliminated here? Is it only needed for development work?

Reply:
I don't know why it's in the environment.yml. It's not in our [conda-forge feedstock](https://github.com/conda-forge/ale-feedstock/blob/master/recipe/meta.yaml) so I think I'll take it out of the environment.yml here.